doc/static_route/SONiC_static_route_hdl.md

```diff
@@ -0,0 +1,546 @@
+
+|Test Cases (done on default instance and data VRF)| Test Result |
+| :------ | :----- |
+|create route for a prefix with NH IP addresses|    verify the route is created|
+|create route for a prefix with NH IP address and interface| verify route is created|
+|create route for a prefix with interface only| verify route is created|
+|create route for a prefix with NH IP addresses and NH VRF|    verify the route is created|
+|create route for a prefix with NH IP address, interface and NH VRF| verify route is created|
+|create route for a prefix with interface and NH VRF| verify route is created|
+|create route for a prefix with NH IP addresses and distance|    verify the route is created|
+|create route for a prefix with NH IP address, interface and distance | verify route is created|
+|create route for a prefix with interface and distance | verify route is created|
+|create route for a prefix with NH IP addresses, NH VRF and distane |    verify the route is created|
+|create route for a prefix with NH IP address, interface and distance| verify route is created|
+|create route for a prefix with interface, NH VRF and distance | verify route is created|
+|create route for a prefix with blackhole and distance |verify route is created
+|CLI: show ip route static|   verify static routes are displayed with flag "S" (i.e Static)|
+|Delete route for a prefix with NH IP addresses| verify the route is deleted|
+|Delete route for a prefix with NH IP address and interface| verify route is deleted|
+|Delete route for a prefix with interface only| verify route is deleted|
+|Delete route for a prefix with NH IP addresses and NH VRF|    verify the route is deleted|
+|Delete route for a prefix with NH IP address, interface and NH VRF| verify route is deleted|
+|Delete route for a prefix with interface and NH VRF| verify route is deleted|
+|Delete route for a prefix with blackhole| verify route is deleted|
+
+| Test cases for management static routes |Test Result |
+| :------ | :----- |
+|create route in default instance  for a prefix with NH IP address and interface as Management0 | verify route is created|
+|create route in default instance  for a prefix with interface as Management0 | verify route is created|
+|create route in mgmt vrf  for a prefix with NH IP address and interface as in-band interface | verify route is created|
+|create route in mgmt vrf  for a prefix with NH IP address | verify route is created|
+|create route in mgmt vrf  for a prefix with in-band interface | verify route is created|
+|create route in data VRF for a prefix with NH IP addresses and NH VRF as management vrf|    verify the route is created|
+|create route in data VRF for a prefix with NH interface and NH VRF as management vrf|    verify the route is created|
+|create route in data VRF for a prefix with NH interface as Management0 and NH VRF as default|    verify the route is created|

```

Comment:
Is this a valid route creation ? should not NH VRF and interface belong to the same VRF ?

Reply:
this is when no mgmt  VRF is created and eth0 (mgmt interface) is part of default VRF.